.changesets/docs_igni_docs_link_update.md

```diff
@@ -0,0 +1,6 @@
+### Update links in docs 
+
+Some of our documentation links were not quite up to date.
+While the links were not broken (the former links redirect to the new url), this changeset updates the links to prevent the extra hop.
```

Comment:
```suggestion
Some of our documentation links were pointing to pages which have been renamed and received new page names during routine documentation updates.  While the links were not broken (the former links redirected to the new URLs) we've updated them to avoid the extra hop
```